homeassistant/components/frontier_silicon/media_player.py

```diff
@@ -266,16 +282,18 @@ def is_volume_muted(self):
     async def async_volume_up(self):
         """Send volume up command."""
         volume = await self.fs_device.get_volume()
-        await self.fs_device.set_volume(volume + 1)
+        await self.fs_device.set_volume(
+            min(int(volume or 0) + 1, self._volume_steps)
```

Comment:
Extract some of it into a variable to make it more readable:
```python
volume = int(volume or 0) + 1
await self.fs_device.set_volume(
            min(volume, self._volume_steps)
)
```

Reply:
Done